### DISCUSSION TITLE
Schedule for other things such as bus and library

### DISCUSSION THREAD

**Discussion by 2217441** [iiumschedule/iium_schedule#89] [Ideas]
- [ ] Academic calendar
- [ ] Important dates
- [ ] Bus schedule
- [ ] Library schedule
- [ ] Swimming Pool schedule
- [ ] Offices schedule
- [ ]

and other things I can't remember right now.
Okay tak?

**Comment by iqfareez** [comment]
Hi @2217441, thanks for the idea. 😄

However, I think I would like to keep the scope of the app small, for now. Let me know if you're thinking otherwise. For the items you've mentioned above, I think it would fit another app.

There are a few I'm aware right now:
- IIUM Mobile: https://play.google.com/store/apps/details?id=my.edu.iium.iiummobile
- Studa: https://play.google.com/store/apps/details?id=com.dev.kusyuk.studentreport

But, I'm not sure how well they are maintained.

**Reply by 2217441** [reply]
I checked both of them and I think they could be improved by being open-source so that future students can reproduce and maintain them. It would also be nice if the schedules were offline or integrated. 

I like your project and I want to contribute when possible, but I’m still learning. 😅
